index.html

```diff
@@ -0,0 +1,204 @@
+          <div class="card flex-sm-row flex-row flex-lg-column ">
+            <img src="./imgs/react-redux.jpg" class="card-img-top" alt="react-redux">
+            <div class="card-body">
+              <h5 class="card-title">ReactJS & Redux</h5>
+              <p class="card-text">
+                We will teach you one of popular frameworks ReactJS and Redux to handle states
+                to make your work easer and professional
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-3 col-md-6 col-12 mt-2">
+          <div class="card flex-sm-row flex-row flex-lg-column ">
+            <img src="./imgs/node-mongo.png" class="card-img-top" alt="Ruby & Ruby on rails">
+            <div class="card-body pt-0">
+              <h5 class="card-title">Ruby On Rails</h5>
+              <p class="card-text">
+                Ruby on Rails makes it much easier and more fun. 
+                It includes everything you need to build fantastic applications,
+              </p>
+            </div>
+          </div>
+        </div>
+        <a href="#" class="h4 mt-5">
+          SEE WHOLE PROGRAM
+        </a>
+      </div>
+    </section>
+    <!-- Start Featured Teachers -->
+    <section class="title featured-teachers container text-center">
+      <h2 class="mb-5 mt-5">
+        Featured Teachers
+      </h2>
+      <div id="teachers" class="row">
+
+      </div>
+    </section>
+    <!-- Start Partner -->
+    <section class="partner container-fluid py-4 mb-5">
+      <h2 class="title text-center pt-3 mb-5">
+        Partner
+      </h2>
+      <div class="d-flex flex-wrap justify-content-around align-items-center">
+        <div class="img-container mt-2">
+          <img src="./imgs/mozilla.png" alt="mozilla" class="w-100"/>
+        </div>
+        <div class="img-container mt-2">
+          <img src="./imgs/line-friends.png" alt="line-friends" class="w-100"/>
+        </div>
+        <div class="img-container mt-2">
+          <img src="./imgs/googlekorea.png" alt="googlekorea" class="w-100"/>
+        </div>
+        <div class="img-container mt-2">
+          <img src="./imgs/airbnb.png" alt="airbnb" class="w-100"/>
+        </div>
+      </div>
+    </section>
+    <!-- Start Footer -->
+    <footer class="container-fluid py-2">
+      <div class="row justify-content-sm-around justify-content-center align-items-center">
+        <img src="./imgs/logo1.jpg" alt="logo" class="col-lg-3 col-sm-3 col-7 my-4">
+        <p class="text-dark col-lg-6 col-sm-7 col-9">
+          We are a community of teachers and students. We work togther to create creative people who can
+          make awesome websites. So we have a special curriculum to make you professional web developer.
+          We will teach you the basics which are html, css, and js. After that you will learn framework for css
+          which is bootstrap 5. and a framework for js which is reactJS.
+        </p>
+      </div>
+    </footer>
```

Comment:
- [ ] This image is too big, please reduce it and follow the original design. Check the layout and background color too.

Your design:

![imagen](https://user-images.githubusercontent.com/60631456/132546433-c1362dcb-3553-48f8-93d8-76f2667b898d.png)

Original design:

![imagen](https://user-images.githubusercontent.com/60631456/132546510-9524816d-b6f8-4d09-973c-8b8e4f79e317.png)

Reply:
Nailed this!